source/light/contracts/Light.sol

```diff
@@ -146,6 +171,25 @@ contract Light is ILight {
     );
   }
 
+  function setFee(uint256 newFee) external onlyOwner {
+    fee = newFee;
```

Comment:
require(fee < X)

Reply:
thanks, yes that is 100% supposed to be there 👍 will finish this PR today